docs/faq.md

```diff
@@ -116,4 +116,64 @@ cite_assay@meta.features <- cite_row_metadata
 
 # add altExp from SingleCellExperiment as second assay to Seurat
 seurat_object[["CITE"]] <- cite_assay
-```
\ No newline at end of file
+```
+
+## What if I want to use Python instead of R? 
+
+Gene expression data files are only available as RDS files, and must be opened in R to view the contents. 
+If you prefer to work in python, count data can first be converted into the 10X format using [`DropletUtils::write10xCounts()`](https://rdrr.io/bioc/DropletUtils/man/write10xCounts.html). 
+Note that you will need to install the [`DropletUtils` package](https://www.bioconductor.org/packages/devel/bioc/html/DropletUtils.html) to use this function.
+
+This will output three files to a new directory:
+- the counts matrix in sparse matrix format - `matrix.mtx.gz`
+- the row names, or gene names, saved as a TSV - `features.tsv.gz`
+- the column names, or cell barcodes, saved as a TSV - `barcodes.tsv.gz`
+
+```
+library(SingleCellExperiment)
+
+# read in the RDS file to be converted
+sce <- readRDS("SCPCL000000_filtered.rds")
+
+# extract the counts matrix to be saved 
+rna_counts <- counts(sce)
+
+# write counts to 10X format and save to a folder named "SCPCL000000-rna"
+DropletUtils::write10xCounts("SCPCL000000-rna", rna_counts, 
+                             barcodes = colnames(rna_counts),
+                             gene.id = rownames(rna_counts))
+
+```
+
+If a library has associated CITE-seq that exists, you will have to save that separately.
+
+```
+# extract the CITE counts matrix to be saved 
+cite_counts <- counts(altExp(sce))
+
+# write counts to 10X format
+DropletUtils::write10xCounts("SCPCL000000-cite", cite_counts, 
+                             barcodes = colnames(cite_counts),
+                             gene.id = rownames(cite_counts))
+
```

Comment:
Same suggestion for simplification as above, but here I am adding the version 3 optional info that denotes this as antibody tags. If we want to do this, we should add `version = "3"` above as well.  (It looks like we _could_ combine the outputs into a single matrix, but I feel like keeping them separate still seems easier to understand & keep track of). 
```suggestion
# write CITE-seq counts to 10X format
DropletUtils::write10xCounts("SCPCL000000-cite", counts(altExp(sce)), 
                             barcodes = colnames(altExp(sce)),
                             gene.id = rownames(altExp(sce)),
                             gene.type = "Antibody Capture",
                             version = "3")

```